assets/styles/main.less

```diff
@@ -1,3 +1,4 @@
+// Automatically injected wiredep dependencies (never manually edit this block)
```

Comment:
Automatically injected bower dependencies via wiredep (never manually edit this block)